packages/data-service-generator/src/tests/__snapshots__/create-data-service-custom-path.spec.ts.snap

```diff
@@ -3465,14 +3469,14 @@ class Customer {
 
   @ApiProperty({
     required: false,
-    type: Number,
+    type: Decimal,
   })
-  @IsNumber()
+  @IsDecimal()
```

Comment:
why can't we keep using the current @IsNumber validator? 
if we decide we need something specific for Decimal, why don't we also use it for Float?